docs/user-guides/pallet-proxy.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 16
+---
```

Comment:
Overall impression is that you have the text to explain features on a high level, but I'm not sure if this would be sufficient for user to start using proxies.

Some thing should be explained in more depth, and some things should be better visualized.
Comparing it to [Polkadot's doc](https://wiki.polkadot.network/docs/learn-proxies) or [Moonbeam's doc](https://docs.moonbeam.network/tokens/manage/proxy-accounts/) we are lacking.

You can provide links to the official Polkadot docs to supplement info provided on this page (e.g. for pure proxies).

The guideline should be - if user comes to this page, reads it - will they need to google proxy accounts again and go read about them on Polkadot's or Moonbeam's page, or is the info here sufficient?